src/backend/api/Fusion.Resources.Api/Notifications/InternalRequestNotification/NotifyTaskOwnerHandler.cs

```diff
@@ -26,7 +26,7 @@ protected override async Task Handle(NotifyTaskOwner request, CancellationToken
             {
                 var allocationRequest = await GetInternalRequestAsync(request.RequestId);
 
-                var arguments = new NotificationArguments($"A personnel request has been updated") { AppKey = "personnel-allocation" };
+                var arguments = new NotificationArguments($"A personnel request has been updated (you are notified as task owner)") { AppKey = "personnel-allocation" };
```

Comment:
This one is maybe a bit harder to word correctly, but perhaps something along the lines of: A personnel request for which you are the task owner has been updated